recipe/recipe.yaml

```
@@ -1,43 +1,31 @@
 context:
   name: "siscone"
-  version: "3.0.6"
+  version: "3.1.0"
 
 package:
   name: ${{ name }}
   version: ${{ version }}
 
 source:
   url: https://gitlab.com/fastjet/siscone/-/archive/siscone-${{ version }}/siscone-siscone-${{ version }}.tar.gz
-  sha256: fd852e5ae3349a1ad540e49b1dd7efe3a188c3f7591902de4aeb781c421ddf13
-  patches: 0001-remove-os-libtoolize-assumptions.patch
+  sha256: bb168b404d8c1e487ac3193ac497b0026dfab7d01037e299a97a14d865b378e4
 
 build:
-  number: 2
+  number: 0
   script:
     # Get an updated config.sub and config.guess
-    - cp $BUILD_PREFIX/share/gnuconfig/config.* .
-    - autoreconf --install
-    - ./configure --help
-    - bash autogen.sh --prefix=$PREFIX --disable-static
-    - make --jobs="${CPU_COUNT}"
-    - if: build_platform == target_platform
-      then: make check
-    - make install
-    - make clean
-  skip:
-    - win
+    - cmake -S . -B build -DCMAKE_INSTALL_PREFIX=${PREFIX}
```

```suggestion
    - cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=${PREFIX} -S . -B build
    - cmake -LH build
```

Ah you were faster than me. So this is updated suggestion for `CAMKE_ARGS` and  `cmake -LH build`.